configs/centernet/metafile.yml

```diff
@@ -0,0 +1,39 @@
+Collections:
+  - Name: CenterNet
+    Metadata:
+      Training Data: COCO
+      Training Techniques:
+        - SGD with Momentum
+        - Weight Decay
+      Training Resources: 8x NVIDIA V100 GPUs
+      Architecture:
+        - ResNet
+    Paper: https://arxiv.org/abs/1904.07850
+    README: configs/centernet/README.md
+
+Models:
+  - Name: centernet_resnet18_dcnv2_140e_coco
+    In Collection: CenterNet
+    Config: configs/centernet/centernet_resnet18_dcnv2_140e_coco.py
+    Metadata:
+      Training Memory (GB): 3.47
```

Comment:
Should add `Batch Size` in Metadata. It would be better to add `inference time (s/im)` if you tested the speed.